### ISSUE TITLE
"Rebuild Core Library" appears in the project menu for all projects, not just imported sketches

### ISSUE THREAD

**Issue by b-perkins** [chipKIT32/chipKIT-importer#11]
After briefly discussing this with Larry it seems like it's pretty minor issue, since selecting rebuild-core-library option in a normal MPLABX project it doesn't appear to do anything harmful (or anything at all). 

I just wanted to post it here as a minor issue for future versions.

FYI - this was plugin version 0.9.11 with MPLABX 3.55.07

Thanks,
Brent

**Comment by gholdys** [comment]
Good catch! The "rebuild core library" action is a so called "context aware action" which means that it should appear only for certain type of projects. The constructor of this action checks the selected project to see if it's root directory contains a "chipKIT-core" directory. Only if this directory exists, the action should be visible. Unfortunately the action was declared as "lazy" which for some reason prevents this mechanism from working properly and the action appears for all projects. I will make this action not lazy and that will fix the problem.
Committed a change to the action class.
